### DISCUSSION TITLE
want to enter wifi configuration mode without button... what to do?

### DISCUSSION THREAD

**Discussion by MechatronikMonkey** [arendst/Tasmota#21957] [Q&A]
I configured a few options, custom template and wifi calibration and uploaded my own berry script.
Now - for testing purposes - I want to bring this prototype device to a new location / WiFi. 

I want to only remove wifi credentials and that the device entering wifi configuration mode on startup to reconfigure wifi. But all the rest should stay the same. Regredfuly there is no button on my prototype and disassembly is not possible at this point. 

What should I do?

**Comment by sfromis** [comment]
Well, simply do what you suggested, remove the Wifi credentials. Then Tasmota should restart in Wifi Manager mode.

`Backlog SSId1 0;Password1 0`
